process.py

```diff
@@ -90,7 +90,7 @@ def report_time():
                 "Calculating costs for %s - %s", time_period.name, ", ".join(modes)
             )
             cost_settings = cost.CostSettings(
-                server_url="http://localhost:8080",
+                server_url="http://" + config.hostname + ":" + str(config.port),
```

Comment:
Have you considered any url validation? 

What would occur if a config hostname of "http://my_hostname" was provided by mistake